website

nandika123 opened the pull request.

YashJipkate commented on celebrations/Design.htm:
```diff
@@ -0,0 +1,112 @@
+            <img src="pics/w2.jpg" width="250" height="202">
+            <img src="pics/w3.jpg" width="250" height="202">
+            <h2> Baby Showers</h2>
+            <img src="pics/baby,jpg.jpg" width="275" height="183">
```
Never have special characters other than underscores in filenames.

YashJipkate commented on celebrations/preloader.js:
```diff
@@ -0,0 +1,7 @@
+});$(window).on('load', function() {
+    $('.preloader-wrapper').fadeOut('slow');
+    $('body').removeClass('preloader-site');
+});
```
This is too similar to @KRT2305 's submission [here](https://github.com/COPS-IITBHU/CSOC-dev-week2/pull/1). Your preloaders are misaligned too, and surprisingly to same extent. Have you both done it together? @KRT2305 Could you confirm (since your PR was earlier than this)?

KRT2305 commented:
Yes Sir,
Her Preloader was going on forever and the deadline was nearing so she used
mine.

And regarding the mis-alignments I'll make sure next time there are none.
Actually this was my first try at web dev.


On Sat, Jun 8, 2019, 11:44 Yash Jipkate <notifications@github.com> wrote:

> *@YashJipkate* commented on this pull request.
>
> @nandika123 <https://github.com/nandika123> Generally looks good. Try to
> improve on the UI though. You can use Bootstrap which is an easy to use
> beauty library. I see you have just included its CDN but haven't used it
> (correct me if I'm mistaken here please).
>
> You could also try using different fonts to make the webpage look good.
> ------------------------------
>
> In celebrations/Design.htm
> <https://github.com/COPS-IITBHU/CSOC-dev-week2/pull/2#discussion_r291795905>
> :
>
> > +        <h1>DESIGN & DECOR</h1>
> +        <p>
> +            Now, have a look at our decorations from some of our previous parties.
> +        </p>
> +        <div id= "bday">
> +            <h2> Birthday Parties</h2>
> +            <img src="pics/birthday.jpg" width="225" height="224">
> +            <img src="pics/b2.jpg" width="275" height="183">
> +            <img src="pics/b3.jpg" width="275" height="184">
> +            <br>
> +            <h2> Weddings</h2>
> +            <img src="pics/w1.jpg" width="266" height="190">
> +            <img src="pics/w2.jpg" width="250" height="202">
> +            <img src="pics/w3.jpg" width="250" height="202">
> +            <h2> Baby Showers</h2>
> +            <img src="pics/baby,jpg.jpg" width="275" height="183">
>
> Never have special characters other than underscores in filenames.
> ------------------------------
>
> In celebrations/preloader.js
> <https://github.com/COPS-IITBHU/CSOC-dev-week2/pull/2#discussion_r291795984>
> :
>
> > @@ -0,0 +1,7 @@
> +$(window).on('DOMContentLoaded', function() {
> +    var Body = $('body');
> +    Body.addClass('preloader-site');
> +});$(window).on('load', function() {
> +    $('.preloader-wrapper').fadeOut('slow');
> +    $('body').removeClass('preloader-site');
> +});
>
> This is too similar to @KRT2305 <https://github.com/KRT2305> 's
> submission here <https://github.com/COPS-IITBHU/CSOC-dev-week2/pull/1>.
> Your preloaders are misaligned too, and surprisingly to same extent. Have
> you both done it together? @KRT2305 <https://github.com/KRT2305> Could
> you confirm (since your PR was earlier than this)?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/COPS-IITBHU/CSOC-dev-week2/pull/2?email_source=notifications&email_token=AKTGG5OKCHPMYFTVMQ2WEYDPZNE45A5CNFSM4HVGRGCKYY3PNVWWK3TUL52HS4DFWFIHK3DMKJSXC5LFON2FEZLWNFSXPKTDN5WW2ZLOORPWSZGOB27F2KA#pullrequestreview-247356712>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AKTGG5OWZ47P4KMD5FSZFLDPZNE45ANCNFSM4HVGRGCA>
> .
>

YashJipkate commented:
@KRT2305 This is not something that should be promoted. The main aim of these assignments are to make you familiar to the basics and so you learn something. 

But anyway, we don't mind a helping friend! :P

Happy to merge once @nandika123 confirms this themselves and addresses the review comments.

Thanks!

nandika123 commented:
Yes, I could not resolve the problem with the preloader. So I took his help
as the deadline was approaching.

On Sun, 9 Jun, 2019, 12:03 PM Yash Jipkate, <notifications@github.com>
wrote:

> @KRT2305 <https://github.com/KRT2305> This is not something that should
> be promoted. The main aim of these assignments are to make you familiar to
> the basics and so you learn something.
>
> But anyway, we don't mind a helping friend! :P
>
> Happy to merge once @nandika123 <https://github.com/nandika123> confirms
> this themselves and addresses the review comments.
>
> Thanks!
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/COPS-IITBHU/CSOC-dev-week2/pull/2?email_source=notifications&email_token=AKTIWEKFOUKZ5OZOMGTPOI3PZSP4ZA5CNFSM4HVGRGCKYY3PNVWWK3TUL52HS4DFVREXG43VMVBW63LNMVXHJKTDN5WW2ZLOORPWSZGODXIEODI#issuecomment-500188941>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AKTIWEMDBMCTFXRZZUQKZ4LPZSP4ZANCNFSM4HVGRGCA>
> .
>

YashJipkate commented:
Thanks @nandika123 As long as you both acknowledge, this is not plagiarism. But refrain from doing this in future. Try to debug it yourselves since those skills are more important.

Merging!
